src/lib/WeatherVane/weathervane_params.c

```diff
@@ -44,8 +44,12 @@
  *
  * @boolean
  * @group Multicopter Position Control
+ * @min 0
+ * @max 3
+ * @bit 0 Enabled.
```

Comment:
I'm wondering if this isn't exactly obvious that "Enabled" covers everything except for takeoff. Any input @hamishwillee?

Reply:
@RomanBapst I prefer "Enable" over "Enabled". 
1. I assume that if bit 0 is unset, it doesn't matter what the other bits are set to? i.e. to have on takeoff you must have **both** bit 0 and 1?
2. You say this is active for all position control modes by default, except takeoff. I assume that means it is also active for landing and loiter by default?

Assuming that, how about:
```
/**
 * Enable weathervane and configure use in optional flight modes.
 *
 * When enabled, weathervane is active for most position control modes.
 * The additional bits can be used to enable weathervane for the remaining modes.
 * 
 * @boolean
 * @group Multicopter Position Control
 * @min 0
 * @max 3
 * @bit 0 Enable.
 * @bit 1 Enable for auto takeoff.
 */
PARAM_DEFINE_INT32(WV_CONFIG, 3);
```

We will also need to update https://docs.px4.io/en/config_vtol/vtol_weathervane.html